tests/httpwrappers/tests.py

```diff
@@ -531,6 +531,50 @@ def test_content_type_mutually_exclusive(self):
             )
 
 
+def test_text_property(self):
+    test_cases = [
+        {
+            "content": "Hello, world!",
+            "expected_text": "Hello, world!",
+            "message": "Initial content test failed.",
+        },
+        {
+            "content": "Updated content",
+            "expected_text": "Updated content",
+            "message": "Content update test failed.",
+        },
+        {
+            "content": b"Ol\xc3\xa1 Mundo",
+            "expected_text": "Olá Mundo",
+            "charset": "utf-8",
+            "message": "UTF-8 encoding test failed.",
+        },
+        {
+            "content": b"Ol\xe1 Mundo",
+            "expected_text": "Olá Mundo",
+            "charset": "iso-8859-1",
+            "message": "ISO-8859-1 encoding test failed.",
+        },
+        {
+            "content": "Cached content",
+            "expected_text": "Cached content",
+            "message": "Caching test (initial) failed.",
+        },
+        {
+            "content": "New content",
+            "expected_text": "New content",
+            "message": "Caching test (after update) failed.",
+        },
+    ]
+
+    for case in test_cases:
+        with self.subTest(case=case):
```

Comment:
Providing the whole dict as the subTest label strikes me as too noisy: it duplicates the failure message and makes it harder to pick out the content. I'd probably just use the content value?

Reply:
I understand the method you suggested, and I agree with your point. I will revise that part as you advised. Thank you! :)